src/Profiles/LEV/DataPages/RX/ANTPLUS_LevSpeedSysinfo1.cpp

```diff
@@ -0,0 +1,36 @@
+#include <Profiles/Lev/DataPages/RX/ANTPLUS_LevSpeedSysinfo1.h>
+#include <Profiles/Lev/ANTPLUS_LevPrivateDefines.h>
+
+LevSpeedSysinfo1::LevSpeedSysinfo1(AntRxDataResponse& dp) : LevBaseMainDataPage(dp) {
+	return;
+}
+
+uint8_t LevSpeedSysinfo1::getTempState()
+{
+	return getData(1);
+}
+
+uint8_t LevSpeedSysinfo1::getTravelModeState()
+{
+	return getData(2);
+}
+
+uint8_t LevSpeedSysinfo1::getSystemState()
+{
+	return getData(3);
+}
+
+uint8_t LevSpeedSysinfo1::getGearState()
+{
+	return getData(4);
+}
+
+uint8_t LevSpeedSysinfo1::getError()
```

Comment:
Lets use getErrorMessage, might get confused with getError in the framework.

Reply:
Done